DougieDev/whats-cookin-dougie-rachel
# Rw refactor event listener

#### What changes/additions are a part of this PR?
- Refactored event listener as discussed from just 1 on the page body to 5 event listeners, divided into sections of the site 
- Refactored the search function to work on keyup to dynamically populate as you suggested 
- Did some minor re-ordering

#### Any background context you want to provide?
None needed
#### Where should the reviewer start?
Scripts.js - query selectors & event listeners towards top
#### How should this be tested (test suite/manually)?
Ensure nothing broke on the site by running index.html. I did test it, but it's good to double-check. 
#### What are the relevant tickets?
Ticket #62 
#### Screenshots (if helpful)
#### Any questions/additional comments:
